tests/comparison.test.ts

```diff
@@ -1,22 +1,27 @@
 import fs from "fs"
 import CoreHandler from "../src/index"
 import extractExceptionsFromAho from "./helpers/extractExceptionsFromAho"
-import generateMockPncQueryResult from "./helpers/generateMockPncQueryResult"
+import generateMockPncQueryResultFromAho from "./helpers/generateMockPncQueryResultFromAho"
 import MockPncGateway from "./helpers/MockPncGateway"
 import processTestFile from "./helpers/processTestFile"
 
 const filePath = "test-data/e2e-comparison"
 
-const tests = fs
+let tests = fs
   .readdirSync(filePath)
   .map((name) => `${filePath}/${name}`)
   .map(processTestFile)
   .filter((t) => !t.incomingMessage.match(/<br7:HearingOutcome/))
 
+const filter = process.env.FILTER_TEST
+if (filter) {
+  tests = tests.filter((t) => t.file.includes(filter))
+}
+
```

Comment:
Can tests be a const and then include this append this filter to line 15? This feels a little odd.

Reply:
`.filter((t) => process.env.FILTER_TEST ? t.file.includes(filter) : true)`